library/core/src/intrinsics.rs

```diff
@@ -2393,3 +2417,63 @@ where
 {
     called_in_const.call_once(arg)
 }
+
+#[cfg(bootstrap)]
+pub(crate) const unsafe fn assert_validity_of<T>(_: *const T) -> bool {
+    true
+}
+
+#[cfg(not(bootstrap))]
+/// Asserts that the value at `value` is valid at type T.
```

Comment:
Is it valid to say "valid at type T"?

Reply:
No idea, but it's what i'm used to saying. I guess "is a valid T" is a less awkward way of saying it. I'll change it to say that.